.github/workflows/browser-tests.yml

```diff
@@ -62,6 +57,7 @@ env:
     APP_ENV: behat
     APP_DEBUG: 1
     PHP_INI_ENV_memory_limit: 512M
+    COMPOSER_CACHE_DIR: /home/runner/.composer/cache
```

Comment:
Non-Github, local runners might be better of using `~/.composer/cache`. Otherwise we rely on an absolute path which may or may not exist / be possible to create.

Reply:
Good idea, let me quickly test it

Reply:
Added in https://github.com/ibexa/gh-workflows/pull/5/commits/d7ff134639306c9a8aad2dbefd95ffc97e9350b8, thanks!